packages/graphql/tests/schema/comments.test.ts

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { printSchemaWithDirectives } from "@graphql-tools/utils";
+import { lexicographicSortSchema } from "graphql/utilities";
+import { Neo4jGraphQL } from "../../src";
+
+describe("Comments", () => {
+    test("Simple", () => {
+        const typeDefs = `
+"A custom scalar."
+scalar CustomScalar
+
+"An enumeration of movie genres."
+enum Genre {
+    ACTION
+    DRAMA
+    ROMANCE
+}
+
+"""
+A type describing a movie.
+"""
+type Movie {
+    id: ID
+    "The number of actors who acted in the movie."
+    actorCount: Int
+    """
+    The average rating for the movie.
+    """
+    averageRating: Float
+    """
+    Is the movie active?
+
+    This is measured based on annual profit.
+    """
+    isActive: Boolean
+    genre: Genre
+    customScalar: CustomScalar
+}`;
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+        const printedSchema = printSchemaWithDirectives(lexicographicSortSchema(neoSchema.schema));
+
+        expect(printedSchema).toMatchInlineSnapshot(`
+            "schema {
+              query: Query
+              mutation: Mutation
+            }
+
+            type CreateInfo {
+              bookmark: String
+              nodesCreated: Int!
+              relationshipsCreated: Int!
+            }
+
+            type CreateMoviesMutationResponse {
+              info: CreateInfo!
+              movies: [Movie!]!
+            }
+
+            \\"\\"\\"A custom scalar.\\"\\"\\"
```

Comment:
Are the escape characters needed here? 🤔

Reply:
yeah, those snapshots are generated - so whatever's there is correct :)